SMLHelper/Utility/AudioUtils.cs

```diff
@@ -42,5 +54,55 @@ public static Channel PlaySound(Sound sound)
             RuntimeManager.LowlevelSystem.playSound(sound, channels, false, out Channel channel);
             return channel;
         }
+
+        /// <summary>
+        /// The a list the different volume controls in the game
+        /// </summary>
+        public enum VolumeControl
```

Comment:
Move the `VolumeControl` enum out of the `AudioUtils` class.
Also, I think the name `VolumeChannel` would fit it a bit better than the current one.

Reply:
The newly renamed `SoundChannel` enum is still inside the `AudioUtils` class.  
Put it into its own file in the same folder and namespace.

Reply:
I know, but it doesn't really make sense to put it outside of the AudioUtils class, since it's only used in there.

Reply:
It's a lot cleaner to use `SoundChannel.Master` instead of `AudioUtils.SoundChannel.Master` every time.